### PR TITLE
chore(docs): add a updated liveness error modal example

### PR DESCRIPTION
**Pull request by thaddmt**
#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        </Flex>
+      </Flex>
+    );
+  }, [error]);
+
+  return (
+    <FaceLivenessDetector
+      sessionId={'sessionId'}
+      region={'us-east-1'}
+      onAnalysisComplete={async () => {}}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:34]: It's kind of strange to me to provide a noop here, does it really need to be a required prop if it doesn't need to do anything?

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      onError={(error) => {
+        setError(error);
+      }}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:37]: Think this can just be:
```suggestion
      onError={setError}
```

Unless TS gets upset 😅

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          justifyContent="center"
+          padding="32px"
+        >
+          <Heading color="black">{error?.state}</Heading>
+          <Text>{error?.error.message}</Text>
+          <Button>Try again?</Button>
+        </Flex>
+      </Flex>
+    );
+  }, [error]);
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:28]: Realize this is just an example, but to reduce componeny complexity and save on memoization memory overhead prefer to declare this component outside of `CustomizationComponentsErrorView` and just pass in `errorHeading` and `errorMessage` props (do not feel strongly abt those names)

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        height="100%"
+      >
+        <Flex
+          backgroundColor="white"
+          direction="column"
+          justifyContent="center"
+          padding="32px"
+        >
+          <Heading color="black">{error?.state}</Heading>
+          <Text>{error?.error.message}</Text>
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:23]: Can we rename the local `error` variable to something like `livenessError` to avoid accessing `message` via `error.error`?

File: docs/src/pages/_document.page.tsx [hunk 5]
```diff
@@ -28,7 +28,7 @@ const getCSPContent = (context: Readonly<DocumentProps>) => {
       img-src 'self' blob: ${ANALYTICS_CSP.all.img.join(' ')};
       connect-src 'self' *.shortbread.aws.dev ${ANALYTICS_CSP.all.connect.join(
         ' '
-      )} https://*.algolia.net https://*.algolianet.com https://cdn.jsdelivr.net https://tfhub.dev https://storage.googleapis.com;
+      )} https://*.algolia.net https://*.algolianet.com https://cdn.jsdelivr.net https://tfhub.dev https://storage.googleapis.com https://cdn.liveness.rekognition.amazonaws.com;
```
Comment (calebpollman) [inline docs/src/pages/_document.page.tsx:31]: Is this change related to the example?

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        alignItems="center"
+        width="100%"
+        height="100%"
+      >
+        <Flex
+          backgroundColor="white"
+          direction="column"
+          justifyContent="center"
+          padding="32px"
+        >
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:21]: Can these be combined in to one `Flex` element?

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+          <Heading color="black">{error?.state}</Heading>
+          <Text>{error?.error.message}</Text>
+          <Button>Try again?</Button>
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:24]: Should clicking the button clear the error?

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponents.tsx [hunk 8]
```diff
@@ -23,6 +23,7 @@ export function CustomizationComponents() {
           );
         },
         ErrorView: ({ children }) => {
+          // children contains a default error component based on the LivenessError, you can use our default rendered components or render your own error message
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponents.tsx:26]: Prefer to add this line in the docs above the example to prevent drift between the example code and the docs themselves

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+        </Flex>
+      </Flex>
+    );
+  }, [error]);
+
+  return (
+    <FaceLivenessDetector
+      sessionId={'sessionId'}
+      region={'us-east-1'}
```
Comment (calebpollman) [inline docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:33]: More of a style thing that should be enforced via linting/prettier, but prefer double quotes for string props:
```suggestion
      sessionId="sessionId"
      region="us-east-1"
```

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx (hunk 6 above)
Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:21]: Hm I did it this way so we could have the outer flex element take up the full component width/height and we could center the inner flex element. This way developer can more easily control positioning of their inner element wherever they want in the camer view

File: docs/src/pages/_document.page.tsx (hunk 5 above)
Comment (thaddmt) [reply docs/src/pages/_document.page.tsx:31]: Not exactly related but is a needed change, will make it in a separate pr

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx (hunk 7 above)
Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx:24]: I'll add a comment here that they should handle the onclick themselves and re-render the component with a new session id

File: docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponents.tsx (hunk 8 above)
Comment (thaddmt) [reply docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponents.tsx:26]: Will do

**Review by esauerbo — DISMISSED** [review]

**Review by reesscot — APPROVED** [review]

**Review by hbuchel — APPROVED** [review]
